lib/sass/calcite-web/patterns/_accordion.scss

```diff
@@ -5,11 +5,11 @@
 //  ↳ patterns → _accordion.md
 
 @mixin accordion() {
-  @include side-nav();
 }
+
   @mixin accordion-section() {
     &.is-active {
-      .accordion-icon {
+      .accordion-icon svg {
```

Comment:
Was something too specific here or why did you have to add the element selector?

Reply:
hmmm. When I remove `svg` arrow does not rotate. Is there a better way to approach this?

Reply:
we could also do something like this: https://github.com/julio8a/calcite-web/compare/components/accordion...paulcpederson:accordion?expand=1

just give the icon a class directly, the wrapping span serves no purpose

Reply:
Sweet, merged it into my branch.

Reply:
This example also makes the icons correct (outline style, 16px) and tweaks the type size so that it matches Bryans designs.